README.md

```diff
@@ -43,6 +43,18 @@ Congrats, you're all set!
 
 **TIP:** We recommend using [NVM](https://github.com/creationix/nvm) to ensure you have a compatible Node.js version (>= 8.11.1).
 
+### Commits
+⚠️**Important:** All commits must adhere to the [Coventional Commits format](https://www.conventionalcommits.org/), because we have an [automated release process](https://semantic-release.gitbook.io/semantic-release/) that depends on this.
+
+While you can, in principle, manually format your commit messages using your preferred git client, we recommend using [commitizen](http://commitizen.github.io/cz-cli/), which is installed when you run `npm install`. To run it, simply do:
+
+```sh
+npm run commit
+```
+(Alternatively, you can also run `npx git-cz`, which does the same thing)
```

Comment:
Actually npx rebuilds the entire thing and runs a potentially separate version from our local install. Might not be a good idea to run it this way.

Reply:
Oh! I did not know that. Will fix before merging.